docs/source/user-guide/general/deploying_on_ec2.rst

```diff
@@ -0,0 +1,11 @@
+====================
+Deploying on AWS EC2
+====================
+
```

Comment:
In order to deploy our open source version on Amazon AWS, you will need to have a Linux server with the following minimum requirements and then follow the installation instructions here: https://docs.deepchecks.com/monitoring/stable/installation/self_host.html